.appveyor.yml

```diff
@@ -27,10 +27,10 @@ install:
   - ps: >-
       if (!(Test-Path -Path C:\vim)){
         Add-Type -A System.IO.Compression.FileSystem
-        Invoke-WebRequest ftp://ftp.vim.org/pub/vim/pc/vim80-586w32.zip `
+        Invoke-WebRequest ftp://ftp.vim.org/pub/vim/pc/vim90w32.zip `
```

Comment:
I'd like to keep the Windows Vim tests on Vim 8.0, so we test the lowest supported version there.

Reply:
I'll revert this file and if CI passes I'll generate the Dockerfile and get this in.